config/install/field.field.node.event.field_event_description.yml

```diff
@@ -1,3 +1,4 @@
+uuid: 943f8f85-0fee-4c24-813c-23b02e3f9bd0
```

Comment:
This should be removed. The UUID key shouldn't be present in module install config.